### PR TITLE
fix: add configuration.json placeholders

### PR DESCRIPTION
**Pull request by nikhname**
Will create amplifyconfiguration.json and awsconfiguration.json at build time in amplify-tools android plugin

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-app/src/index.js [hunk 1]
```diff
@@ -356,9 +356,22 @@ async function createAndroidHelperFiles() {
   };
   const configJsonStr = JSON.stringify(configJsonObj, null, 4);
   const configFile = path.join(process.cwd(), './amplify-gradle-config.json');
+  const emptyJsonStr = JSON.stringify({});
+  const rawPath = path.join(process.cwd(), './app/src/main/res/raw');
```
Comment (kaustavghosh06) [inline packages/amplify-app/src/index.js]: Will this work in windows?

File: packages/amplify-app/src/index.js [hunk 2]
```diff
@@ -356,9 +356,22 @@ async function createAndroidHelperFiles() {
...
+  const awsConfigFile = path.join(rawPath, './awsconfiguration.json');
```
Comment (kaustavghosh06) [inline packages/amplify-app/src/index.js]: same as above^

File: packages/amplify-app/src/index.js (hunk 1 above)
Comment (nikhname) [reply packages/amplify-app/src/index.js]: Updated

File: packages/amplify-app/src/index.js (hunk 2 above)
Comment (nikhname) [reply packages/amplify-app/src/index.js]: updated

**Pull request merged**